apps/src/music/views/ExemplarPlayerView.tsx

```diff
@@ -30,15 +30,26 @@ const ExemplarPlayerView: React.FunctionComponent<ExemplarPlayerViewProps> = ({
   const dispatch = useAppDispatch();
   const isPlaying = useAppSelector(state => state.music.isPlaying);
   const [exemplarIsPlaying, setExemplarIsPlaying] = useState<boolean>(false);
+  const analyticsReporter = useRef<AnalyticsReporter>(new AnalyticsReporter());
 
   const onMount = useCallback(async () => {
     await player.preloadSounds(playbackEvents, () => {});
+
+    analyticsReporter.current?.startSession();
```

Comment:
We shouldn't need to start a new session here because the ExemplarPlayerView is always used within the context of a Music Lab level (right?). The analytics session would have been started by the level already.